modules/installation-adding-registry-pull-secret.adoc

```diff
@@ -103,18 +102,20 @@ $ mkdir -p <directory_name>
 ----
 +
 Where `<directory_name>` is either `~/.docker` or `$XDG_RUNTIME_DIR/containers`.
+
 .. Copy the pull secret to the appropriate directory by entering the following command:
```

Comment:
I cannot find this sentence in any of the preview links.

Reply:
I see that; I looked through the other previewed links and didn't find it, either.. I don't think oc-mirror is a distro...I wonder if this should be an ifeval for "about-installing-oc-mirror-v2.doc"? @snarayan-redhat WDYT? Any insight is greatly appreciated.

Reply:
IIUC, `ifeval` is already placed on the top the file - Line 18 onward.

```
ifeval::["{context}" == "installing-mirroring-disconnected"]
:restricted:
:oc-mirror:
endif::[]

ifeval::["{context}" == "about-installing-oc-mirror-v2"]
:oc-mirror-v2:
:restricted:
endif::[]
```

You can find this step [here](https://102851--ocpdocs-pr.netlify.app/openshift-enterprise/latest/disconnected/installing-mirroring-disconnected#installation-adding-registry-pull-secret_installing-mirroring-disconnected). This doesn't apply to `Updating` docs, only appears in oc-mirror docs.